vision/setup.py

```diff
@@ -48,9 +48,10 @@ def create_version_file():
     'Pillow<=6.2.1',
     'numpy>=1.16.0',
     'matplotlib',
-    'gluoncv>=0.5.0,<1.0',
+    'gluoncv==0.9.0b20201125',
     'graphviz<0.9.0,>=0.8.1',
     'pandas>=1.0.0,<2.0',
+    'd8>=0.0.2',
```

Comment:
Please cap version such as <0.1 or <1 , since its possible API could break given it is so new.